MeesGame/Gameplay/GameManagement/GameEnvironment.cs

```diff
@@ -55,6 +57,11 @@ public static GameSettingsManager GameSettingsManager
         get { return gameSettingsManager; }
     }
 
+    public static GameEnvironment GetGameEnvironment
```

Comment:
The normal way to handle singletons is to call it Instance. Then you can use GameEnvironment.Instance. So I suggest renaming it to Instance